.github/workflows/check.yaml

```diff
@@ -50,15 +50,16 @@ jobs:
             git diff
             exit 1
           fi
-  Staticcheck:
+  GolangCI-Lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4.2.2
         with:
           fetch-depth: 1
-      - uses: dominikh/staticcheck-action@v1.3.1
+      - uses: golangci/golangci-lint-action@v8.0.0
         with:
-          version: "2022.1.1"
+          version: latest
+          args: --timeout=5m
```

Comment:
This argument is not needed as the same timeout setting is present in the configuration of `golangci-lint`.